man/gnt-instance.rst

```diff
@@ -178,10 +178,10 @@ name
    identifier. An instance can not have two NICs with the same name.
 
 vlan
-   in openvswitch mode specifies the VLANs that the NIC will be
-   connected to. To connect as an access port use ``n`` or ``.n`` with
-   **n** being the VLAN ID. To connect as an trunk port use ``:n[:n]``.
-   A hybrid port can be created with ``.n:n[:n]``
+   in openvswitch and bridged mode specifies the VLANs that the NIC
+   will be connected to. To connect as an access port use ``n`` or 
+   ``.n`` with **n** being the VLAN ID. To connect as an trunk port
+   use ``:n[:n]``. A hybrid port can be created with ``.n:n[:n]``
```

Comment:
One final remark: we should add a note that for this to work, the bridge must already have VLAN filtering enabled. Other than that, LGTM.